template/package.json

```diff
@@ -17,9 +17,9 @@
     "@babel/core": "^7.12.9",
     "@babel/runtime": "^7.12.5",
     "@react-native-community/eslint-config": "^3.0.0",
-    "babel-jest": "^26.6.3",
+    "babel-jest": "^29.0.3",
```

Comment:
I just released 29.2.0 fwiw. But I guess these versions mirror some internal version and should remain in sync with that?

Reply:
yeah I want to keep the versioning consistent across the entire repo

Reply:
Keeping them consistent makes sense, but I would like to bump all of them to at least 29.1.0 before 0.71, because `jest.now()` is important for properly mocking (legacy) fake timer animations. 

I'll pull this in and then have a go at bumping internally across the monorepo. Should be smooth, right? 🤞

Reply:
hopefully! 😀